monkey/agent_plugins/payloads/ransomware/config-schema.json

```diff
@@ -5,33 +5,33 @@
             "type": "string",
             "pattern": "^(\\.[A-Za-z0-9_]+)?$",
             "description": "The file extension that the Infection Monkey will use for the encrypted file.",
-            "default": ".m0nk3y",
```

Comment:
I wonder how this ever worked? Maybe whatever we used to evaluate these is very tolerant?

Reply:
I am not really sure but good thing we got it.